code-for-canada/project-thundercat
# Privacy notice links

# Description

Add links to the privacy notice and fix french off by one error in content.

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)


## Screenshot
![image](https://user-images.githubusercontent.com/4640747/60114245-713cf700-9741-11e9-8f1a-73e6f0e4eb3d.png)


# Testing
Manual steps to reproduce this functionality:

1.  Go to account creation page.
2.  Click the link to see the privacy dialog.

# Checklist

Applicable for all code changes.
